qa/workunits/rbd/rbd_support_module_recovery.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -ex
+
+POOL=rbd
+IMAGE_PREFIX=image
+NUM_IMAGES=20
+RUN_TIME=3600
+
+rbd mirror pool enable ${POOL} image
+rbd mirror pool peer add ${POOL} dummy
+
+# Create images and schedule their mirror snapshots
+for ((i = 1; i <= ${NUM_IMAGES}; i++)); do
+    rbd create --size 1G ${POOL}/${IMAGE_PREFIX}$i --image-feature exclusive-lock
```

Comment:
Nit: `--size` has a short variant, group options before positional arguments
```suggestion
    rbd create -s 1G --image-feature exclusive-lock ${POOL}/${IMAGE_PREFIX}$i
```